lib/Source.js

```diff
@@ -29,15 +41,7 @@ export default class Source extends Node {
     this.password = password
```

Comment:
`this.user` and `this.password` are still needed?

Reply:
Good point. It is being used in `LookupSource#fromSource` and `CubeSource#fromSource` but can be made redundant.